mirage/serializers/internal-resource.ts

```diff
@@ -11,7 +10,7 @@ export default class InternalResourceSerializer extends ApplicationSerializer<In
             configuredStorageAddons: {
                 links: {
                     related: {
-                        href: `${addonServiceAPIUrl}internal_resources/${model.id}/configured_storage_addons`,
+                        href: `${addonServiceAPIUrl}internal-resources/${model.id}/configured-storage-addons`,
```

Comment:
```suggestion
                        href: `${addonServiceAPIUrl}internal-resources/${model.id}/configured-storage-addons/`,
```
to match https://github.com/CenterForOpenScience/ember-osf-web/pull/2107/files#diff-34e59843696a4ce1ffea7d2707d97d08851e7ab33b4958854d6412e56b47ecd3R538 ?